app/controllers/houston/dashboards/staging_controller.rb

```diff
@@ -1,6 +1,6 @@
 class Houston::Dashboards::StagingController < ApplicationController
   layout "houston/dashboards/dashboard"
-  STAGING_PROJECTS = %w(members unite ledger bsb)
+  STAGING_PROJECTS = %w(members unite ledger bsb members-dav)
```

Comment:
Better yet, instead of making the API call to GitHub, Houston already syncs pull requests with GitHub, so you can just check `Github::PullRequest`:

```ruby
Github::PullRequest.find_all { |pr| pr.labels.member?("on-staging") }
```

Or, with a small change to houston-core,

```ruby
Github::PullRequest.labeled "on-staging"
```